llvm/include/llvm/CodeGen/GlobalISel/LegalizerInfo.h

```diff
@@ -1198,6 +1197,11 @@ class LegalizeRuleSet {
     assert(MinTy.getElementType() == MaxTy.getElementType() &&
            "Expected element types to agree");
 
+    if (MinTy.isScalableVector())
+      return actionIf(LegalizeAction::Unsupported, always);
+    if (MaxTy.isScalableVector())
+      return actionIf(LegalizeAction::Unsupported, always);
```

Comment:
This isn't used/needed yet? It should probably just be an assert with the current interfaces. We will need to find a way to fix those in the future.